### PR TITLE
Gok 211-This PR handles the Save functionality in stockReceipt tab.

### PR DESCRIPTION
**Pull request by deepthi-mantena**

### CODE REVIEW

**Comment by gsluthra** [comment]
Please provide some description on what all tasks this code does and if possible, also provide a screenshot. thanks!

**Comment by deepthi-mantena** [comment]
@gsluthra As part of this PR I have modified save-receipt.js stock-receipt.jsx and apiUtils.js to handle save by constructing the request body by making respective api calls and populating the values of Requestbody attributes.The below screen shot shows the basic landing page which would display on click of stockfetch button after entering an outwardID
Screenshot 2023-07-24 at 9 46 01 PM
Now I updated Batchquantity to 5 
Screenshot 2023-07-24 at 9 51 21 PM
Now in Db these values shown On Ui should be saved in appropriate columns.

Screenshot 2023-07-24 at 9 52 58 PM
Screenshot 2023-07-24 at 9 54 29 PM

File: src/service/save-receipt.js [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
 // eslint-disable-next-line import/prefer-default-export
-export const saveReceipt = async (data) => {
-  console.log('data', data)
+
+   const saveReceipt = async (items,outwardNumber,destinationUuid) => {
+
+       const instanceTypeResponse = await getRequest(`/openmrs/ws/rest/v2/inventory/stockOperationType?v=full&q=Receipt`);
+       const instanceTypeUuids = instanceTypeResponse.results[0].uuid;
+       const currentDate = new Date();
+       const formattedDate = `${currentDate.getDate().toString().padStart(2, '0')}-${
```
Comment (gsluthra) [inline src/service/save-receipt.js]: Ideally should be in a date-utils.js. And should have a unit test.

Comment (deepthi-mantena) [reply src/service/save-receipt.js]: @gsluthra as suggested moved date formatting to dateutils.js and updated PR .For unittest may be we will create a tech debt card.

**Review by gsluthra — APPROVED** [review]

File: src/utils/date-utils.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const getFormattedDate = () => {
```
Comment (gsluthra) [inline src/utils/date-utils.js:3]: Add a comment which explains what does this method do. thanks!

Comment (deepthi-mantena) [reply src/utils/date-utils.js:3]: Yes @gsluthra added the comments.

**Pull request merged**